core/modules/qdisp/testQDisp.cc

```diff
@@ -266,23 +266,11 @@ BOOST_AUTO_TEST_CASE(QueryResource) {
     r->ProvisionDone(NULL);
     BOOST_CHECK(status.getInfo().state  == qdisp::ExecStatus::PROVISION_NACK);
     char buf[20];
-    strcpy(buf, qdisp::XrdSsiSessionMock::getMockString(false));
-    qdisp::XrdSsiSessionMock xsMockFalse(buf);
-    r = new qdisp::QueryResource(s.resource.path(), s.request, s.requester,
-            qdisp::Executive::newNotifier(ex, refNum),
-            retryTest, status);
-    r->ProvisionDone(&xsMockFalse);
-    BOOST_CHECK(status.getInfo().state  == qdisp::ExecStatus::REQUEST_ERROR);
-    BOOST_CHECK(retryTest->_retryCalled == true);
-    // At this point, r has been deleted by QueryResource::ProvisionDone
-
-    LOGF_INFO("QueryResource test 2");
-    retryTest->_retryCalled = false;
+    strcpy(buf, qdisp::XrdSsiSessionMock::getMockString(true));
```

Comment:
Should getMockString lose it's boolean argument? I'm not sure it is ever false now.